python/unicornafl/__init__.py

```diff
@@ -228,9 +180,25 @@ def uc_afl_fuzz_custom(uc: Uc,
     cb3 = ctypes.cast(UC_AFL_FUZZ_CALLBACK_CB(
         _fuzz_callback_cb), UC_AFL_FUZZ_CALLBACK_CB)
 
-    err = _uc2afl.uc_afl_fuzz_custom(uc._uch, input_file.encode("utf-8"), cb1, cb3, 
-        cb2, always_validate, persistent_iters, ctypes.cast(idx, ctypes.c_void_p))
-
+    if isinstance(input_file, str):
+        input_file = input_file.encode('utf-8')
+    if isinstance(input_file, bytes):
+        input_file = ctypes.cast(ctypes.c_char_p(input_file), ctypes.c_void_p).value
+    elif input_file is None:
+        input_file = 0
+    else:
+        raise TypeError("Input file should be string or bytes or None")
+    err = uc_afl_fuzz_custom_impl(
+        uc._uch.value,
+        input_file,
+        ctypes.cast(cb1, ctypes.c_void_p).value,
+        ctypes.cast(cb3, ctypes.c_void_p).value,
+        ctypes.cast(cb2, ctypes.c_void_p).value,
+        always_validate,
+        persistent_iters,
+        ctypes.cast(idx, ctypes.c_void_p)
```

Comment:
Sorry for intruding, but is it possible that is should be this:
```suggestion
        ctypes.cast(idx, ctypes.c_void_p).value
```

Reply:
Nice catch. Thanks.